runtime/vanilla/Cargo.toml

```diff
@@ -111,12 +111,11 @@ runtime-benchmarks = [
   'frame-benchmarking',
   'frame-support/runtime-benchmarks',
   'frame-system-benchmarking',
-  'hex-literal',
```

Comment:
shall we remove this line? I'm not clear on this

Reply:
should be removed while it's not an optional requirement.